libraries/Microsoft.Bot.Builder.Dialogs.Debugging/DebuggingBotAdapterExtensions.cs

```diff
@@ -2,11 +2,21 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Runtime.CompilerServices;
 using Microsoft.Bot.Builder.Dialogs.Debugging.CodeModels;
 using Microsoft.Extensions.Logging;
 
+#if SIGNASSEMBLY
```

Comment:
We can probably deffer adding this once we have actual unit tests for these classes. 

if we do add it, I would put it in a file called AssemblyInfo.cs (old school I know) but it is easier to spot.